pkg/cli/cli.go

```diff
@@ -181,18 +181,24 @@ func Shell() {
 				case "reload":
 					menuSetModule(strings.TrimSuffix(strings.Join(shellModule.Path, "/"), ".json"))
 				case "run":
+					var m string
 					r, err := shellModule.Run()
 					if err != nil {
 						message("warn", err.Error())
+						break
+					}
+					if shellModule.Lang == "Go" {
+						m, err = agents.AddJob(shellModule.Agent, r[0], r[1:])
 					} else {
-						m, err := agents.AddJob(shellModule.Agent, "cmd", r)
-						if err != nil {
-							message("warn", err.Error())
-						} else {
-							message("note", fmt.Sprintf("Created job %s for agent %s at %s",
-								m, shellModule.Agent, time.Now().UTC().Format(time.RFC3339)))
-						}
+						m, err = agents.AddJob(shellModule.Agent, "cmd", r)
+					}
+					if err != nil {
+						message("warn", err.Error())
```

Comment:
Can you add a little context to the error message so that if it is triggered, the user has additional context?